internal/config/config.go

```diff
@@ -19,28 +19,33 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-type Tokens struct {
-	GithubToken string `yaml:"githubToken"`
-	JiraToken   string `yaml:"jiraToken"`
+type Config struct {
+	JiraBaseURL string `yaml:"jiraBaseURL"`
+	AuthTokens  struct {
+		JiraToken   string `yaml:"jira"`
+		GithubToken string `yaml:"github"`
+	} `yaml:"authTokens"`
```

Comment:
nit: my preference is to separate these and nest them.   However, experimenting today I found that you cannot do that with yaml tags, and can only do so with json meta tags. 

For e.g. https://github.com/grokspawn/gh2jira/blob/9ed979fa11fe0fb6cbfb3295b537b908ed9406a5/internal/config/config.go#L25